torchgeo/datasets/oscd.py

```diff
@@ -375,13 +379,15 @@ def __init__(
             batch_size: The batch size to use in all created DataLoaders
             num_workers: The number of workers to use in all created DataLoaders
             val_split_pct: What percentage of the dataset to use as a validation set
+            crop_size: Size of random crop from image and mask (n, height, width)
```

Comment:
I think you should split crop_size up into something like `num_patches_per_tile` and `patch_size`, currently it is unintuitive (maybe just to me!) to have the number of samples in something called `crop_size`

Reply:
Yeah I was also unsure of this, just wanted to keep the # of args down but maybe it would be worth it to split it up in this case.